packages/plugins/color-picker/jest.config.front.js

```diff
@@ -2,5 +2,8 @@
 
 module.exports = {
   preset: '../../../jest-preset.front.js',
+  transform: {
+    '^.+\\.ts(x)?$': ['@swc/jest'],
```

Comment:
do you think this could just go straight in the root `jest-preset.front.js`?

Reply:
It broke when I tried for the BE so I didn't try here. I'll check 👍